buildkite/client/pipeline.go

```diff
@@ -85,56 +93,32 @@ func (c *Client) CreatePipeline(pipeline *Pipeline) (*Pipeline, error) {
 	return &result, nil
 }
 
-// createPipelineGraphQl will create the pipeline but only set the required fields
-// after creation, UpdatePipeline will be used to set the rest of the fields via
-// the REST API
-func (c *Client) createPipelineGraphQl(pipeline *Pipeline) (*Pipeline, error) {
-	req := graphql.NewRequest(`
-mutation PipelineCreateRequest($pipelineCreateInput: PipelineCreateInput!) {
-  pipelineCreate(input: $pipelineCreateInput) {
-    pipeline {
-      slug
-    }
-  }
-}`)
-
-	orgID, err := c.GetOrganizationId(c.orgSlug)
-	if err != nil {
-		return nil, err
-	}
-
-	req.Var("pipelineCreateInput", map[string]interface{}{
-		"organizationId": orgID,
-		"name":           pipeline.Name,
-		"repository": map[string]string{
-			"url": pipeline.Repository,
-		},
-		"steps": map[string]string{
-			"yaml": pipeline.Configuration,
+// createYAMLPipeline will create the pipeline but only set the required fields
+// filled with stubs. After creation, UpdatePipeline will be used to set the rest
+// of the fields via the REST API
+func (c *Client) createYAMLPipeline(pipeline *Pipeline) (*Pipeline, error) {
+	created, err := c.CreatePipeline(&Pipeline{
```

Comment:
Previously, it was not possible to modify `configuration` via the GraphQL API if the pipeline was created via the REST API. Has this changed recently? Migrations from the REST API to GraphQL needed to be done through the undocumented Website API.

What scenarios have you tested?